### ISSUE TITLE
Open ODT in documents app instead of ODT viewer

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/documents#204]
We can pretty much phase out the ODT viewer now, can’t we? ODT documents should always be opened inside the Documents app when clicked from Files.

cc @VicDeo @DeepDiver1975

**Comment by christophknoth** [comment]
I found it also very confusing that I cannot edit .odt files immediately and a lot of my collaborators as well. How complicated would it be to have this as an option? Or can I overwrite this myself somehow?

**Comment by jancborchardt** [comment]
Yeah – @VicDeo can we finally switch that around? Neither LibreOffice nor Word nor Pages have something such as a read-only mode.

**Comment by jancborchardt** [comment]
(Btw cheers @christophknoth, cool to see you in here. :)

**Comment by VicDeo** [comment]
@jancborchardt PRs are welcome.
BTW scroll 3 comments above and you'll see readonly LibreOffice

**Comment by jancborchardt** [comment]
@VicDeo you are the maintainer of this app, right? And I’m not a coder. So »PRs are welcome« is a really strange answer – I know that, but we should solve this issue for 9.0 because it’s breaking the flow.

**Comment by VicDeo** [comment]
@jancborchardt ok, i'm a coder. Will do, if I have time for that.

**Comment by jancborchardt** [comment]
Thanks! :)
